Doc/library/itertools.rst

```diff
@@ -691,25 +691,36 @@ loops that truncate the stream.
 
         def tee(iterable, n=2):
             if n < 0:
-                raise ValueError('n must be >= 0')
-            iterator = iter(iterable)
-            shared_link = [None, None]
-            return tuple(_tee(iterator, shared_link) for _ in range(n))
-
-        def _tee(iterator, link):
-            try:
-                while True:
-                    if link[1] is None:
-                        link[0] = next(iterator)
-                        link[1] = [None, None]
-                    value, link = link
-                    yield value
-            except StopIteration:
-                return
-
-   Once a :func:`tee` has been created, the original *iterable* should not be
-   used anywhere else; otherwise, the *iterable* could get advanced without
-   the tee objects being informed.
+                raise ValueError
```

Comment:
Is there any particular reason to remove the text for this exception? Not that it matters much, but the C code does retain the text.

Reply:
I thought it was distracting and would take maintenance effort to keep it synced with the C code.  Also, in a pure python equivalent, the surrounding code is sufficiently informative.